client/my-sites/customer-home/cards/actions/quick-links/index.jsx

```diff
@@ -235,7 +235,9 @@ export const QuickLinks = ( {
 						<>
 							<ActionBox
 								hideLinkIndicator
-								gridicon="plans"
+								iconComponent={
+									<JetpackLogo monochrome className="quick-links__action-box-icon" />
+								}
```

Comment:
Using the same Jetpack logo as other jetpack-related quick links, instead of a slightly different jetpack logo